docs/content/api/java-api.md

```diff
@@ -592,10 +592,22 @@ public class StreamWriteTable {
 
         // 3. Collect all CommitMessages to a global node and commit
         StreamTableCommit commit = writeBuilder.newCommit();
-        commit.commit(commitIdentifier, messages);
-
-        // 4. When failover, you can use 'filterCommitted' to filter committed commits.
-        commit.filterCommitted(committedIdentifiers);
+        boolean firstTry = true;
```

Comment:
Here is very confused...
Actually, when failure occurs, just `filterAndCommit` messages in state, and use commit for new messages.